zkevm-circuits/src/keccak_circuit/test.rs

```diff
@@ -16,17 +26,82 @@ fn verify<F: Field>(k: u32, inputs: Vec<Vec<u8>>, success: bool) {
         }
         panic!();
     }
+
+    // Extract the content of the lookup table.
+    let hashes = {
```

Comment:
is the naming hashes a bit confusing? suggest something like hash_lookup_entry / hash_lookup_tuple etc

Reply:
done